From: =?utf-8?q?Sebastian_Dr=C3=B6ge?= <slomo@debian.org>
Date: Tue, 13 Jul 2010 14:06:28 +0200
Subject: Adjust path to glib-compile-schemas in the pkg-config file

This is because gio-querymodules and glib-compile-schemas have been put in
a private, versioned directory in libglib2.0-0 to avoid a dependency loop.

Forwarded: not-needed, specific to Debian packaging
---
 gio-2.0.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio-2.0.pc.in b/gio-2.0.pc.in
index afd3a40..cd024fe 100644
--- a/gio-2.0.pc.in
+++ b/gio-2.0.pc.in
@@ -7,7 +7,7 @@ datadir=@datadir@
 schemasdir=${datadir}/glib-2.0/schemas
 bindir=@bindir@
 giomoduledir=@GIO_MODULE_DIR@
-glib_compile_schemas=${bindir}/glib-compile-schemas
+glib_compile_schemas=@libdir@/glib-2.0/glib-compile-schemas
 glib_compile_resources=${bindir}/glib-compile-resources
 gdbus_codegen=${bindir}/gdbus-codegen
 
